tracetree.h

```diff
@@ -76,7 +76,7 @@ class TraceTree
             if (it == parent->children.end() || it->instructionPointer != ip) {
                 index = m_index++;
                 it = parent->children.insert(it, {ip, index, {}});
-                fprintf(out, "t %" PRIxPTR " %x\n", reinterpret_cast<uintptr_t>(ip), parent->index);
```

Comment:
%p is printed as 0xABCDEF but we only want ABCDEF without the leading 0x. PRIxPTR is what we want, and that is also the accepted answer of the stackoverflow question you are linking to?